src/visualizations/config/generators/dhis/singleValue.js

```diff
@@ -106,7 +106,8 @@ const generateDashboardItem = (config, legendSet) => {
             config.value,
             config.formattedValue,
             config.subText,
-            legendSet
+            legendSet,
+            40
```

Comment:
maybe use a constant throughout the file?

Reply:
There are 2 different values, do you mean to have 2 constants each used in one place?

Reply:
So it's just coincidence that both use the number 40?

Reply:
Yes, one is for the text size, the other is for the `y` attribute on the SVG text node to control the spacing with the value above.